### PR TITLE
Homework3

### PR DESCRIPTION
**Pull request by mkravtz**
Refactored Homework 1 to include functions, implement DRY and KISS.

### CODE REVIEW

**Comment by Almantask** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 2
           

Complexity increasing per file
==============================
- Src/BootCamp.Chapter/HwFunctions.cs  1
- Src/BootCamp.Chapter/Program.cs  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=4854476&bid=16104381)

File: Src/BootCamp.Chapter/HwFunctions.cs [hunk 1]
```diff
@@ -0,0 +1,30 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
```
Comment (Almantask) [inline Src/BootCamp.Chapter/HwFunctions.cs]: Codacy Issue found: [Remove this unnecessary 'using'.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=4854476)

File: Src/BootCamp.Chapter/HwFunctions.cs [hunk 2]
```diff
@@ -0,0 +1,30 @@
+﻿using System;
+using System.Collections.Generic;
```
Comment (Almantask) [inline Src/BootCamp.Chapter/HwFunctions.cs]: Codacy Issue found: [Remove this unnecessary 'using'.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=4854476)

**Comment by Almantask** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 3
           

Complexity increasing per file
==============================
- Src/BootCamp.Chapter/HwFunctions.cs  1
- Src/BootCamp.Chapter/Program.cs  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=4854476&bid=16104381)

File: Src/BootCamp.Chapter/HwFunctions.cs [hunk 3]
```diff
@@ -0,0 +1,28 @@
+﻿using System;
+
+namespace BootCamp.Chapter
+{
+    public class HwFunctions
```
Comment (ellmaster) [inline Src/BootCamp.Chapter/HwFunctions.cs]: The name of this class was requested to be Lesson3.

File: Src/BootCamp.Chapter/Program.cs [hunk 4]
```diff
@@ -9,6 +8,18 @@ class Program
...
+            {
+                var surname = HwFunctions.PromptString("Please enter your surname: ");
+                var age = HwFunctions.PromptInt("Please enter your age: ");
+                var weight = HwFunctions.PromptFloat("Please enter your weight (in kg): ");
+                var height = HwFunctions.PromptFloat("Please enter your height (in m): ");
+                Console.WriteLine(surname + " is " + age + " " + "years old, his weight is " + weight + "kg and his height is " + height + "m.");
+                
+                var Bmi = HwFunctions.CalculateBmi(weight, height);
+                Console.WriteLine("Your estimated BMI is " + Bmi);
+            }
```
Comment (ellmaster) [inline Src/BootCamp.Chapter/Program.cs]: That should be the code in the method named "Demo()" of "Lesson3" class.

File: Src/BootCamp.Chapter/HwFunctions.cs (hunk 3 above)
Comment (mkravtz) [reply Src/BootCamp.Chapter/HwFunctions.cs]: Made the changes.

**Comment by Almantask** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 3
           

Complexity increasing per file
==============================
- Src/BootCamp.Chapter/HwFunctions.cs  1
- Src/BootCamp.Chapter/Program.cs  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=4854476&bid=16104381)

**Comment by Almantask** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 3
           

Complexity increasing per file
==============================
- Src/BootCamp.Chapter/HwFunctions.cs  2
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=4854476&bid=16104381)

File: Src/BootCamp.Chapter/Program.cs (hunk 4 above)
Comment (mkravtz) [reply Src/BootCamp.Chapter/Program.cs]: Updated homework to reflect Discord instructions, not the homework instructions from Twitch chat. Thanks!

**Comment by Almantask** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 3
           

Complexity increasing per file
==============================
- Src/BootCamp.Chapter/HwFunctions.cs  2
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=4854476&bid=16104381)

File: Src/BootCamp.Chapter/Checks.cs [hunk 5]
```diff
@@ -16,26 +14,27 @@ public static class Checks
     {
         public static int PromptInt(string message)
         {
-            // To do: call your implementation. 
-            return 0;
+            // To do: call your implementation.
```
Comment (Almantask) [inline Src/BootCamp.Chapter/Checks.cs]: Please remove redundant comments

Comment (mkravtz) [reply Src/BootCamp.Chapter/Checks.cs]: Removed.

**Comment by Almantask** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 3
           

Complexity increasing per file
==============================
- Src/BootCamp.Chapter/HwFunctions.cs  2
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=4854476&bid=16104381)

**Pull request closed without merging**